Dockerfile

```
@@ -1,15 +1,16 @@
 # load the alpine base image
 FROM alpine:3.13
 
-RUN apk add -U python make gcc g++ nodejs yarn npm
+RUN apk add -U python2 make gcc g++ nodejs yarn npm
+ENV PYTHONPATH /usr/lib/python2.7/site-packages
```

test this out -- it should work
```suggestion
RUN apk add -U make gcc g++ nodejs yarn npm
```